src/Timespan.swift

```diff
@@ -27,12 +27,12 @@ THE SOFTWARE.
 */
 public struct TimeSpan
 {
-    private static let secondsPerMinute = 60
-    private static let secondsPerHour = 60 * secondsPerMinute
-    private static let secondsPerDay = 24 * secondsPerHour
+    fileprivate static let secondsPerMinute = 60
+    fileprivate static let secondsPerHour = 60 * secondsPerMinute
```

Comment:
Haha fileprivate is the new `private`!

Private now is waaay more restrictive and is only accessible in the class or struct that it is declared in